chatrooms/serializers.py

```diff
@@ -11,10 +11,14 @@ class Meta:
         fields = ['id','name']
 
 class MessageSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(source='user.username', read_only=True)
+    user = serializers.PrimaryKeyRelatedField(queryset=User.objects.all(), write_only=True)
+   
     class Meta:
         model = Message
-        fields = ['id','chat_romm','user','text']
-
+        fields = ['id','chat_romm','username','text','user']
+    
+    
```

Comment:
Hey Ramesh, all good we will get username in API but why are you adding user . Also, it will be better if we get the name of chatroom instead of chat_romm id. Please insert the name of Chatromm  in API.

Reply:
okay i will make the necessary changes and resubmit the new PR.

Reply:
Okay  this code looks good. All the tests and builds are successfully carried out. I will approve your code and you can merge it.